src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X/test/RazorPageDocumentClassifierPassTest.cs

```diff
@@ -246,7 +247,7 @@ public void RazorPageDocumentClassifierPass_NullFilePath_SetsClass()
         // Assert
         Assert.Equal("global::Microsoft.AspNetCore.Mvc.RazorPages.Page", visitor.Class.BaseType);
         Assert.Equal(new[] { "public" }, visitor.Class.Modifiers);
-        Assert.Equal("AspNetCore_d73aeb762894dcfb9bbaa0e591f8a46be56f978ed801b6f93caf3dfa797acf4e", visitor.Class.ClassName);
+        AssertEx.Equal("AspNetCore_c3b458108610c1a2aa6eede0a5685ede853e036732db515609b2a23ca15359e1", visitor.Class.ClassName);
```

Comment:
Is the diff in the checksums here just the casing on SHA256?

Reply:
Nope. `SourceText` includes the encoder preamble in its checksums, while `RazorSourceDocument` did not.

Reply:
Makes sense.